README.md

```diff
@@ -170,7 +170,7 @@ aliases:
 
     # Allows running the command `cpl setup-app`
     # instead of `cpl apply-template gvc redis postgres memcached rails sidekiq`.
-    setup:
+    setup_app_templates:
```

Comment:
There are more places where we need to update this, such as in `migrating.md` and in the command description. Please search for occurrences of `setup` and replace with `setup_app_templates` wherever appropriate.

Reply:
Done